src/Apps/Partner/Components/PartnerArtists/PartnerArtistsCarousel/PartnerArtistsCarousel.tsx

```diff
@@ -27,25 +27,27 @@ export const PartnerArtistsCarousel: React.FC<PartnerArtistsCarouselProps> = ({
   ) {
     return null
   }
-
-  const artists = extractNodes(partner.artistsConnection)
+  const { slug } = partner
+  const artists = compact(partner.artistsConnection.edges)
 
   return (
     <Shelf alignItems="flex-start">
-      {artists.map(artist => (
-        <CellArtistFragmentContainer
-          key={artist.internalID}
-          artist={artist}
-          to={`/partner/${partner.slug}/artists/${artist.slug}`}
-          FollowButton={
-            <FollowArtistButtonQueryRenderer
-              id={artist.internalID}
-              contextModule={ContextModule.recommendedArtistsRail}
-              size="small"
-            />
-          }
-        />
-      ))}
+      {artists
+        .filter(artist => artist !== null)
```

Comment:
Probably not necessary, but wanted to have extra protection here.

Reply:
`compact` should filter this out up above

Reply:
I have tried calling that on artists before but it was throwing an error because it's not actually an array?
![image](https://user-images.githubusercontent.com/8671643/204926604-c8be6384-8dcd-4391-a8c0-8db6bce0a04d.png)

Reply:
Ah, should do something like 
```tsx
const artists = extractNodes(partner.artistsConnection)
```

I wouldn't worry about compacting or filtering here though. The pattern we typically use is in a comment below

Reply:
so that is what I tried originally but then that doesn't work because the CellPartnerArtistsFragmentContainer needs that fragmentRef 
![image](https://user-images.githubusercontent.com/8671643/205108823-499f3fd2-8666-4604-90ef-c534647b13cd.png)
vs now artists just being fully deconstructed (I think that is the right term here, but not sure) 
![image](https://user-images.githubusercontent.com/8671643/205108992-91a72e59-ae28-4dc8-b43f-5e41ae504655.png)

Reply:
Yep, we're not extracting nodes here since this expects an edge. It's super annoying.

Reply:
If we fixed our connection types to better reflect nullability this wouldn't be an issue.

Reply:
Ah 👍